vivisect/base.py

```diff
@@ -389,7 +397,7 @@ def _handleADDEXPORT(self, einfo):
         self.exports.append(einfo)
         self.exports_by_va[va] = einfo
         fullname = "%s.%s" % (filename,name)
-        self.makeName(va, fullname)
+        self.makeName(va, fullname, makeuniq=True)
```

Comment:
are we married to this idea?  since makeName creates its own event, should this be relegated to the "addExport()" function?  i would very much prefer that, since it would allow for "makeuniq" to live there too, and not be "under cover" as it is now.